vignettes/package.Rmd

```diff
@@ -2,119 +2,250 @@
 title: "Using reticulate in an R Package"
 output:
   rmarkdown::html_vignette
+editor_options:
+  markdown:
+    wrap: 80
 vignette: >
   %\VignetteIndexEntry{Using reticulate in an R Package}
   %\VignetteEngine{knitr::rmarkdown}
   %\VignetteEncoding{UTF-8}
 ---
 
+## Declaring Python Requirements
 
-## Delay Loading Python Modules
+Beginning with Reticulate version 1.41, R packages can declare Python
+requirements with `py_require()`. Reticulate can then resolve these requirements
+at runtime using an ephemeral Python environment.
 
-If you write an R package that wraps one or more Python packages, it's likely that you'll be importing Python modules within the `.onLoad` method of your package so that you can have convenient access to them within the rest of the package source code.
+Calling `py_require()` is always recommended because it has no effect if the
+user is using a manually managed Python environment, making it generally safe to
+use.
```

Comment:
The phrase "Always recommended" feels a bit awkward. Always recommended compared to what?

Reply:
Compared to not calling it? 

I can mention in this paragraph this supersedes previous guidance, like adding a section to DESCRIPTION or calling `use_virtualenv(required = FALSE)` in `.onLoad()`, if that would make it clearer.